doc/examples/cluster_confidence.py

```diff
@@ -0,0 +1,188 @@
+
+# Make a corpus callosum seed mask for tracking
+seed_mask = labels == 2
+seeds = utils.seeds_from_mask(seed_mask, density=[1, 1, 1], affine=affine)
+# Make a streamline bundle model of the corpus callosum ROI connectivity
+streamlines = LocalTracking(csa_peaks, classifier, seeds, affine,
+                            step_size=2)
+streamlines = Streamlines(streamlines)
+
+
+"""
+We do not want our results inflated by short streamlines, so we remove
+streamlines shorter than 40mm prior to calculating the CCI.
+"""
+
+lengths = list(length(streamlines))
+long_streamlines = []
+for i, sl in enumerate(streamlines):
+    if lengths[i] > 40:
+        long_streamlines.append(sl)
+
+
+"""
+Now we calculate the Cluster Confidence Index using the corpus callosum
+streamline bundle and visualize them.
+"""
+
+
+cci = cluster_confidence(long_streamlines)
+
+# Visualize the streamlines, colored by cci
+ren = window.renderer()
+
+hue = [0.5, 1]
+saturation = [0.0, 1.0]
+
+lut_cmap = actor.colormap_lookup_table(scale_range=(cci.min(), cci.max()/4),
+                                       hue_range=hue,
+                                       saturation_range=saturation)
+
+bar3 = actor.scalar_bar(lut_cmap)
+ren.add(bar3)
+
+stream_actor = actor.line(long_streamlines, cci, linewidth=0.1,
+                          lookup_colormap=lut_cmap)
+ren.add(stream_actor)
+
+
+"""
+If you set interactive to True (below), the rendering will pop up in an
+interactive window.
+"""
+
+
+interactive = False
+if interactive:
+    window.show(ren)
+window.record(ren, n_frames=1, out_path='cci_streamlines.png',
+              size=(800, 800))
+
+"""
+.. figure:: cci_streamlines.png
+   :align: center
+
+   Cluster Confidence Index of corpus callosum dataset.
+
+
+If you think of each streamline as a sample of a potential pathway through a
+complex landscape of white matter anatomy probed via water diffusion,
+intuitively we have more confidence that pathways represented by many samples
+(streamlines) reflect a more stable representation of the underlying phenomenon
+we are trying to model (anatomical landscape) than do lone samples.
+
+The CCI provides a voting system where by each streamline (within a set
+tolerance) gets to vote on how much support it lends to. Outlier pathways score
+relatively low on CCI, since they do not have many streamlines voting for them.
+These outliers can be removed by thresholding on the CCI metric.
+
+"""
+
+
+fig, ax = plt.subplots(1)
+ax.hist(cci, bins=100, histtype='step')
+ax.set_xlabel('CCI')
+ax.set_ylabel('# streamlines')
+fig.savefig('cci_histogram.png')
+
+
+"""
+.. figure:: cci_histogram.png
+   :align: center
+
+   Histogram of Cluster Confidence Index values.
+
+Now we threshold the CCI, defining outliers as streamlines that score below 1.
+
+"""
+
+keep_streamlines = []
+for i, sl in enumerate(long_streamlines):
+    if cci[i] >= 1:
+        keep_streamlines.append(sl)
+
+# Visualize the streamlines we kept
+ren = window.renderer()
```

Comment:
Same as above, capital *R* here.  `window.Renderer()` (class object) instead of `window.renderer()` (deprecated function)